supervisor/host/network.py

```
@@ -142,12 +144,22 @@
         self, interface: str, changed: dict[str, Any], invalidated: list[str]
     ):
         """Check if connectivity property has changed."""
+        _LOGGER.info(f"_check_connectivity_changed: {interface} changed {changed} invalidated {invalidated}")
```

Ruff

Logging statement uses f-string

(G004)